packages/components/atoms/f-popover/src/components/Popover.vue

```diff
@@ -2,14 +2,14 @@
     <div
         :class="$style['c-popover']"
         data-test-id="popover">
-        I am a VPopover Component (GB)
+        <slot></slot>
     </div>
 </template>
 
 <script>
 
 export default {
-    name: 'VPopover',
+    name: 'Popover',
```

Comment:
I think this was added as a [styleguide recommendation](https://vuejs.org/v2/style-guide/#Base-component-names-strongly-recommended). Think we need to put a PR in to amend other components to match this format too